pdf/src/campPrint/RichText.vue

```diff
@@ -70,6 +70,23 @@ const rules = [
   {
     shouldProcessNode: (node) => node.type === 'tag' && node.name === 'li',
     processNode: (node, parent) => {
+      const childWithEmptyText = {
+        type: 'text',
+        name: '',
+        attrs: {},
+        children: [],
+        content: '',
+      }
+      if (!node.children.length) {
+        node.children.push({
+          type: 'text',
+          attrs: {},
+          children: [childWithEmptyText],
+        })
```

Comment:
Can't you simplify this?
```suggestion
        node.children.push(childWithEmptyText)
```
The additional child will be added afterwards anyways.

Reply:
node.children[0] also needs a child, so no

Reply:
then both have an empty content.
I just needed to catch both cases: a node with no children, and a node with a child without children.
I did not dive too much into the docs, i just printf'ed what happens when a empty \<li\> appears, and then put that into the implementation

Reply:
done